src/js/discharge-wizard/components/InstructionsPage.jsx

```diff
@@ -2,16 +2,25 @@ import React from 'react';
 import { Link } from 'react-router';
 
 class InstructionsPage extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeFAQ: null,
+    };
+  }
+
   componentDidMount() {
-    const script = document.createElement('script');
-    script.type = 'text/javascript';
-    script.async = true;
-    script.src = 'https://standards.usa.gov/assets/js/vendor/uswds.min.js';
-    script.id = 'uswds';
-    document.getElementsByTagName('head')[0].appendChild(script);
     window.scrollTo(0, 0);
   }
 
+  handleFAQToggle = (e) => {
+    e.preventDefault();
+    this.setState({
+      activeFAQ: this.state.activeFAQ ? null : e.target.name
```

Comment:
If I'm reading this right, it seems like if there's already a question expanded, clicking another question would only close the currently opened question and not open the new one?

Reply:
Do’h you’re right. Need to check whether the question is already open